package.json

```diff
@@ -28,11 +28,15 @@
     "graph:linea-deploy": "graph deploy --node https://graph-deploy.linea.build/ --ipfs https://graph-ipfs.linea.build LedgityLabs/LedgityYield ./subgraph/linea-subgraph.yaml",
     "graph:linea-remove": "graph remove --node https://graph-deploy.linea.build/ LedgityLabs/LedgityYield",
     "graph:arbitrum-gen": "cd subgraph && graph codegen ./arbitrum-subgraph.yaml && graph build ./arbitrum-subgraph.yaml",
-    "graph:arbitrum-deploy": "graph deploy --studio ledgity-yield-arbitrum ./subgraph/arbitrum-subgraph.yaml"
+    "graph:arbitrum-deploy": "graph deploy --studio ledgity-yield-arbitrum ./subgraph/arbitrum-subgraph.yaml",
+    "graph:OKX_X1_testnet-gen": "cd subgraph && graph codegen ./OKX_X1_testnet-subgraph.yaml && graph build ./OKX_X1_testnet-subgraph.yaml",
+    "graph:OKX_X1_testnet-create": "graph create --node https://www.okx.com/api/v1/x1-testnet/index/subgraphs/name/LedgityLabs/LedgityYield LedgityLabs/LedgityYield",
```

Comment:
Are you sure about the value of the --node parameter here?
I would have expected something like `https://www.okx.com/api/v1/x1-testnet/subgraph/`